### PR TITLE
Docs tweaks

### PR DESCRIPTION
**Pull request by benjaoming**
**Work in progress**

* Updates for #4106 and move CONTENT_ROOT note to correct location
* Removed outdated, mis-placed upgrade instructions, we're better off with release notes for people who pay attention to updates than instructing someone about how to upgrade in our offline docs.
* Development environment instructions no longer mis-placed
* Info about building the documentation updated

### CODE REVIEW

**Comment by MCGallaspy** [comment]
Is this still a WIP? @aronasorman you may wish to refer to this as you review the docs.

**Comment by benjaoming** [comment]
I was supposed to add ed's description about the windows command, so that's still missing... but you could add a comment that you've review things thus far, that'll make it easier to look through the last review.

**Comment by benjaoming** [comment]
Actually, just merge it... with all this testing and stuff going on, it's sad that the docs are not as good as possible.

I'll make another PR for additional changes.

File: docs/developer_docs/documentation.rst [hunk 1]
```diff
@@ -1,19 +1,48 @@
 How to contribute to documentation
 ==================================
 
-You can propose changes to the docs directly on github (instructions below) or email your recommendations to info@learningequality.org.
+You can propose changes to the docs directly on Github (instructions below) or
```
Comment (aronasorman) [inline docs/developer_docs/documentation.rst]: Is a newline the only change made here?

File: docs/developer_docs/documentation.rst [hunk 2]
```diff
@@ -1,19 +1,48 @@
...
-* From the base directory, the documenation can be found in the *sphinx-docs* subdirectory. Specific pages of the docs are each associated with a different .rst file, potentially in a subdirectory of *sphinx-docs*.
-* The documentation is written in `ReStructured Text <http://sphinx-doc.org/rest.html>`_ format, so please see the primer!
-* After making your changes, try to build the docs to review them. This process can take some time, as an instance of the server and a browser may need to be started. To build the docs (see *README.md* in the *sphinx-docs* directory for requirements):
+Documentation development
+-------------------------
 
-    * In Linux, if you have the *pyvirtualdisplay* package installed you can build in headless mode by running the command *env SPHINX_SS_USE_PVD=true make html* in the *sphinx-docs* directory.
-    * In other OSes, or to build in non-headless (headed?) mode, just run the command *make html* in the *sphinx-docs* directory. This may launch a browser. Don't interfere with it!
+#. Work from the *develop* branch.
+#. From the base directory, the documenation can be found in the ``docs/``
```
Comment (aronasorman) [inline docs/developer_docs/documentation.rst]: > s/documenation/documentation/

File: docs/usermanual/userman_admin.rst [hunk 3]
```diff
@@ -27,15 +27,6 @@ For users that choose to register online, there are some important terms to fami
 **Web Browser**
     A program that retrieves and presents information resources on the World Wide Web. Popular web browsers include Internet Explorer, Google Chrome, Mozilla Firefox, and Safari.
 
-**Torrent**
```
Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: What's the rationale for removing these definitions?

File: docs/developer_docs/documentation.rst (hunk 1 above)
Comment (benjaoming) [inline docs/developer_docs/documentation.rst]: That, and the capital G for Github :)

File: docs/usermanual/userman_admin.rst (hunk 3 above)
Comment (benjaoming) [inline docs/usermanual/userman_admin.rst]: See jamie's issue.

Comment (benjaoming) [inline docs/usermanual/userman_admin.rst]: #4106

File: docs/developer_docs/documentation.rst (hunk 2 above)
Comment (benjaoming) [inline docs/developer_docs/documentation.rst]: fixed

**Comment by aronasorman** [comment]
A few typos here and there. Should be good to merge.

File: docs/developer_docs/documentation.rst (hunk 1 above)
Comment (radinamatic) [inline docs/developer_docs/documentation.rst]: Maybe then it should be capital G & capital H! ;)

**Pull request merged**